### PR TITLE
Reorder 'TimeUnit' Settings alphabetically.

### PR DESCRIPTION
**Pull request by adambhagemeyer**

### CODE REVIEW

File: src/OpenSEE/Scripts/TSX/defaults.ts [hunk 1]
```diff
@@ -244,12 +244,12 @@ export const defaultSettings = {
     } as OpenSee.IUnitCollection,
     TimeUnit: {
         current: 5, options: [
```
Comment (clackner-gpa) [inline src/OpenSEE/Scripts/TSX/defaults.ts:246]: you may want to change `current` to `0` since it was `auto`

**Review by clackner-gpa — CHANGES_REQUESTED** [review]

File: src/OpenSEE/Scripts/TSX/defaults.ts (hunk 1 above)
Comment (StephenCWills) [reply src/OpenSEE/Scripts/TSX/defaults.ts:246]: That's right. If you don't change this, the default unit for time will be `seconds` instead of `auto`.

Comment (adambhagemeyer) [reply src/OpenSEE/Scripts/TSX/defaults.ts]: Ok, went ahead and fixed it

**Review by StephenCWills — APPROVED** [review]

**Review by clackner-gpa — APPROVED** [review]

**Review by clackner-gpa — APPROVED** [review]

**Review by clackner-gpa — CHANGES_REQUESTED** [review]
We will retarget this to the new openSEE brnach kirsten is working in

**Pull request closed without merging**